SU2_CFD/include/fluid/CSU2TCLib.hpp

```diff
@@ -71,14 +71,30 @@ class CSU2TCLib : public CNEMOGas {
 
   const su2double T_ref_suth = 273.15; /*!<\brief Reference temperature for Sutherland's model [K] */
 
+  /*--- Attractive Coulombic potential constants ---*/
```

Comment:
Should I add a comment with a reference for these values, or okay just to keep track of that in documentation?

Reply:
I think that would be helpful!